modules/host/negotiatesettings.go

```diff
@@ -27,14 +27,21 @@ func (h *Host) managedRPCSettings(conn net.Conn) error {
 	// Set the negotiation deadline.
 	conn.SetDeadline(time.Now().Add(modules.NegotiateSettingsTime))
 
-	h.mu.RLock()
+	h.mu.Lock()
+	h.revisionNumber++
```

Comment:
then you are certain that you didn't forget.

We have a 64bit number of counting available to us, so there's no risk of running out.

The settings are going to be adjusting rapidly either way. The host is going to have bandwidth load balancing at some point, and the bandwidth balancer might update the price once a minute or more. Better to make sure that the settings number is always going up, than to end up in a situation where the host seems to be giving out conflicting settings information.